installer/targets/ros-python-sip/install.bash

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+local SIP_version="4.15.5"
+
+if [ $(lsb_release -sc) == "trusty" ]
+then
+    tue-install-system python-sip-dev
+else
+
+    if command -v sip >/dev/null 2>&1
```

Comment:
Why are you installing python-sip from source?

Reply:
SIP is the 'source' of the problem. Or the combination of KDL with different SIP versions. Therefore we need a specific SIP version.